### PR TITLE
[UXP-20368] - Add support of SWC sp-tabs, sp-tab and sp-tab-panel (v037.0) in swc-uxp-wrapper for UXP

### PR DESCRIPTION
**Pull request by inasirhussain**
Closes - https://jira.corp.adobe.com/browse/UXP-20368

### CODE REVIEW

File: packages/tabs/src/uxp-tab.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+[name='icon'] + #item-label {
+    margin-left: var(
+        --mod-tabs-icon-to-text,
+        var(--spectrum-tabs-icon-to-text)
+    );
+}
+
+:host([disabled]) {
+    color: var(
```
Comment (deepthi-majji) [inline packages/tabs/src/uxp-tab.css:71]: what is the issue with disabled color state in existing styles?

Comment (inasirhussain) [reply packages/tabs/src/uxp-tab.css:71]:
```
::slotted(:not([slot])):focus, ::slotted(:not([slot])):focus-visible {
    color: var(--highcontrast-tabs-color-key-focus, var(--mod-tabs-color-key-focus, var(--spectrum-tabs-color-key-focus))); 
}
```

Above style is conflicting for the disabled state style, so this is required to increase the specificity and hence this gets picked

Comment (deepthi-majji) [reply packages/tabs/src/uxp-tab.css:71]: does SWC not have disabled styles?

Comment (inasirhussain) [reply packages/tabs/src/uxp-tab.css:71]: its there but in UXP somehow, the above style gets the higher specificity hence i had to add disabled styles explicitly with important keyword

**Review by deepthi-majji — APPROVED** [review]

**Comment by deepthi-majji** [comment]
Please resolve the conflicts.

**Pull request merged**